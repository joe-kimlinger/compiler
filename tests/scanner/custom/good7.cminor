// Comparisons
7 && 4;
4 != 4;
5 >= 2;
6 <= 2;
true == false;
true || false;
-!(false)

