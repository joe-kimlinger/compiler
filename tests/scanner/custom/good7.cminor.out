COMMENT
INTEGER_LITERAL 7
AND
INTEGER_LITERAL 4
SEMICOLON
INTEGER_LITERAL 4
NE
INTEGER_LITERAL 4
SEMICOLON
INTEGER_LITERAL 5
GE
INTEGER_LITERAL 2
SEMICOLON
INTEGER_LITERAL 6
LE
INTEGER_LITERAL 2
SEMICOLON
TRUE
EQ
FALSE
SEMICOLON
TRUE
OR
FALSE
SEMICOLON
NEGATION
LPAREN
FALSE
RPAREN
